schemas/HPXMLBaseElements.xsd

```diff
@@ -401,7 +401,7 @@
 					</xs:element>
 					<xs:element maxOccurs="unbounded" minOccurs="0" name="ControlType" type="ClothesDryerControlType"/>
 					<xs:element minOccurs="0" name="Vented" type="HPXMLBoolean"/>
-					<xs:element minOccurs="0" name="VentedFlowRate" type="FlowRate"/>
+					<xs:element minOccurs="0" name="VentedFlowRate" type="HPXMLDoubleGreaterThanOrEqualToZero"/>
```

Comment:
There are pros and cons to replacing these kinds of datatypes (referenced by multiple elements):
- **Pros**: It's now much easier to see directly what the datatype and restrictions are; they are no longer obscured. In fact, I have found a couple questionable datatypes being assigned but it wasn't easy to tell before. There's also less confusion about when to use a generic datatype (like `HPXMLDouble`) vs a more specific datatype (like `FlowRated`).
- **Cons**: When adding, e.g., a new flow rate element in the future, the developer won't be able to simply choose an existing `FlowRate` datatype, rather they'd probably want to look through the schema for similar elements and choose the same datatype. So it requires a bit more brain power.

If the cons outweights the pros, we can consider retaining datatypes like `FlowRate`, `Capacity`, `RValue`, etc., which are reference by multiple elements.

Reply:
I did notice an example where the use of `RValue` for different elements is arguably not appropriate, but the issue was obscured by just pointing to an `RValue` datatape.
- When referring to, say, the perimeter insulation R-value for a slab, allowing zero is fine.
- When referring to a door, an R-value must be greater than zero.